deployments/Dockerfiles/tendermint/app.py

```diff
@@ -82,6 +84,43 @@ def override_config_toml() -> None:
         fp.write(config)
 
 
+def update_peers(validators: List[Dict]) -> None:
+    """Fix peers."""
+
+    config_path = Path(os.environ["TMHOME"]) / "config" / "config.toml"
+    config_text = config_path.read_text(encoding="utf-8")
+
+    new_peer_string = 'persistent_peers = "'
+    for peer in validators:
+        new_peer_string += peer["peer_id"] + "@" + peer["hostname"] + ":" + "26656,"
```

Comment:
This assumes the p2p port is always 26656

Reply:
Yes, this is the issue I was referring. We can introduce a param arg for this.